### DISCUSSION TITLE
Auo Updates

### DISCUSSION THREAD

**Discussion by awptechnologies** [JanDeDobbeleer/oh-my-posh#6041] [General]
When using the auto update config can i just put it at very bottom of theme.json? im still to this day confused where it should go.

**Comment by JanDeDobbeleer** [comment]
@awptechnologies it needs to be added in the root object. So first level right after opening the brackets. Same level as `blocks`.

**Comment by awptechnologies** [comment]
I believe the auto updates are trying to run. Now issue is i get the error that i dont have permissions. What do i need to do to give my local user permissions to run the auto upgrades.

**Reply by JanDeDobbeleer** [reply]
@awptechnologies what do you mean with permissions? If you installed for `allusers`, you won't be able to upgrade unless your run oh-my-posh as admin.

**Comment by awptechnologies** [comment]
Ya that must be my problem. I installed based on a post and it uses sudo to run commands.
